src/assets/i18n/en_US.json

```diff
@@ -888,6 +899,29 @@
   "product.label.sale.text": "SALE",
   "product.label.topseller.text": "TOP",
   "product.manufacturer_name.label": "Manufacturer Name",
+  "product.notification.create.success.message": "Your product notification has been created.",
+  "product.notification.delete.success.message": "Your product notification was deleted.",
+  "product.notification.edit.form.cancel.button.label": "Cancel",
+  "product.notification.edit.form.create.button.label": "OK",
+  "product.notification.edit.form.email.label": "E-mail",
+  "product.notification.edit.form.instock_notification.label": "Notify me when the product is back in stock.",
+  "product.notification.edit.form.no_notification.label": "I no longer wish to receive any notifications about this item.",
+  "product.notification.edit.form.price.error.required": "Please enter a valid amount of money.",
+  "product.notification.edit.form.price.error.valid": "Please enter a valid amount of money.",
+  "product.notification.edit.form.price.label": "Price",
+  "product.notification.edit.form.price_notification.label": "Notify me if the price drops to:",
+  "product.notification.edit.form.update.button.label": "Update",
+  "product.notification.edit.link.title": "Edit notification",
+  "product.notification.price.add_notification.button.label": "Price Notification",
+  "product.notification.price.add_notification.button.title": "Edit notification",
```

Comment:
Key says "add_notification" but translation says "Edit ...". Is this correct here or should it be "Add notification" instead?

Reply:
We adapted the key name.